cram_3d_world/cram_urdf_projection/.gitignore

```diff
@@ -0,0 +1,5 @@
+*.~
```

Comment:
Let's do it like this:

cram_urdf_projection goes to cram_3d_world:

    cram_3d_world/cram_urdf_projection/package.xml

and cram_urdf_bringup goes to somewhere else :) It depends on robot URDFs etc, so definitely shouldn't be in cram_3d_world. For now, I'd say let's put it into cram_pr2 and later find a better place for it once more packages are robot-independent:

    cram_pr2/cram_urdf_bringup/package.xml